examples/solana_native/basic_nft/README.md

```diff
@@ -0,0 +1,423 @@
+     */
+    console.info("+==== Minting... ====+");
+    await mintSendAndConfirm({
+        wallet: johnDoeWallet.publicKey,
+        assocTokenAccount: johnDoeATA,
+        color: "Purple",
+        rarity: "Rare",
+        shortDescription: "Only possible to collect from the lost temple event",
+        signers: {
+            feePayer: feePayer,
+            funding: feePayer,
+            mint: mint,
+            owner: johnDoeWallet,
+        },
+    });
+    console.info("+==== Minted ====+");
+
+    /**
+     * Get the minted token
+     */
+    let mintAccount = await getMint(connection, mint.publicKey);
+    console.info("+==== Mint ====+");
+    console.info(mintAccount);
+
+    /**
+     * Get the Gem Metadata
+     */
+    let gem = await getGemMetadata(gemPub);
+    console.info("+==== Gem Metadata ====+");
+    console.info(gem);
+    console.assert(gem!.assocAccount!.toBase58(), johnDoeATA.toBase58());
+
+    /**
+     * Transfer John Doe's NFT to Jane Doe Wallet (technically, the Associated Token Account)
+     */
+    console.info("+==== Transferring... ====+");
+    await transferSendAndConfirm({
+        wallet: janeDoeWallet.publicKey,
+        assocTokenAccount: janeDoeATA,
+        mint: mint.publicKey,
+        source: johnDoeATA,
+        destination: janeDoeATA,
+        signers: {
+            feePayer: feePayer,
+            funding: feePayer,
+            authority: johnDoeWallet,
+        },
+    });
+    console.info("+==== Transferred ====+");
+
+    /**
+     * Get the minted token
+     */
+    mintAccount = await getMint(connection, mint.publicKey);
+    console.info("+==== Mint ====+");
+    console.info(mintAccount);
+
+    /**
+     * Get the Gem Metadata
+     */
+    gem = await getGemMetadata(gemPub);
+    console.info("+==== Gem Metadata ====+");
+    console.info(gem);
+    console.assert(gem!.assocAccount!.toBase58(), janeDoeATA.toBase58());
+
+    /**
+     * Burn the NFT
+     */
+    console.info("+==== Burning... ====+");
+    await burnSendAndConfirm({
+        mint: mint.publicKey,
+        wallet: janeDoeWallet.publicKey,
+        signers: {
+            feePayer: feePayer,
+            owner: janeDoeWallet,
+        },
+    });
+    console.info("+==== Burned ====+");
+
+    /**
+     * Get the minted token
+     */
+    mintAccount = await getMint(connection, mint.publicKey);
+    console.info("+==== Mint ====+");
+    console.info(mintAccount);
+
+    /**
+     * Get the Gem Metadata
+     */
+    gem = await getGemMetadata(gemPub);
+    console.info("+==== Gem Metadata ====+");
+    console.info(gem);
+    console.assert(typeof gem!.assocAccount, "undefined");
+}
+
+fs.readFile(path.join(os.homedir(), ".config/solana/id.json")).then((file) =>
+    main(Keypair.fromSecretKey(new Uint8Array(JSON.parse(file.toString())))),
+);
+```
+
+Open the terminal, navigate to the `program_client` directory, and execute the following commands:
+
+```shell
+npm install ts-node --save-dev
```

Comment:
Please change the command for `npm install ts-node @solana/spl-token --save-dev`